1.7/administration/installing/custom/system-requirements/index.md

```diff
@@ -107,6 +107,9 @@ Here are the agent node hardware requirements.
 *   Each node has unfettered IP-to-IP connectivity from itself to all nodes in the DC/OS cluster.
 *   UDP must be open for ingress to port 53 on the masters. To attach to a cluster, the Mesos agent node service (`dcos-mesos-slave`) uses this port to find `leader.mesos`. 
 
+### High-Speed Internet Access
+
+High-speed internet access is recommended before installing DC/OS. A minimum 10 MBit per second is required for DC/OS services. The installation of some DC/OS services will fail if the artifact download time exceeds the value of MESOS_EXECUTOR_REGISTRATION_TIMEOUT within the file `/opt/mesosphere/etc/mesos-slave-common`. The default value for MESOS_EXECUTOR_REGISTRATION_TIMEOUT is 10 minutes.
```

Comment:
suggest: "High speed internet access is recommended for DC/OS installation". My thinking is that the word "before" isn't really accurate.